### DISCUSSION TITLE
Negative ADC offset?

### DISCUSSION THREAD

**Discussion by squaretones** [shanteacontrols/OpenDeck#321] [Q&A]
Hi,

I have potentiometers set to send CC messages. When the pot is at its lowest position and I start turning it upward, it takes longer than I would like to produce CC value 1.  I have noticed this across different pots and using both the OpenDeck M board and Raspberry Pi Pico. Ideally for me the pot would send CC value 1 as soon as I start turning it up from its lowest position.

I'm wondering if there is a way to calibrate the board so that it produces CC value 1 as soon as the pot is turned from its lowest position. I think a negative ADC offset might do this, if that is possible?

Thanks for any help or ideas.

**Comment by paradajz** [comment]
There's no way to calibrate that behaviour at the moment. Note that I'm not doing anything special to scale the values, it's 1:1 translated from 10/12 bit range to 7-bit (or 14-bit) range. So no curves or anything like that.
